project_milestone_status/tests/__init__.py

```
@@ -0,0 +1,2 @@
+from . import test_project_milestone_status
```

Ruff (0.8.2)

`.test_project_milestone_status` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)